src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/WindowsRazorProjectHostBase.cs

```diff
@@ -382,7 +382,7 @@ private Task UnconfiguredProject_ProjectRenamingAsync(object? sender, ProjectRen
             }
         }
 
-        path = joinedPath;
```

Comment:
Do we have this issue on Linux? If not, why not?

Reply:
The clue is in the name of the file 😁

Reply:
Well yea, but that doesn't mean this fix doesn't NEED to go in Linux. Just that we did it for Windows 😛

Reply:
The project is the key. `Microsoft.VisualStudio.LanguageServices.Razor` should never be loaded on Linux.

Reply:
We should still verify on Linux /Mac as well.